backend/envconfig/envconfig.go

```diff
@@ -81,3 +83,14 @@ func setFieldValue(field reflect.StructField, fieldValue reflect.Value, newValue
 func NewEnvConfig(environment fw.Environment) EnvConfig {
 	return EnvConfig{environment: environment}
 }
+
+func processInt(value, kind string) (int64, error) {
+	if kind == "int" {
```

Comment:
nit: maybe in the future we can consider switch statement.

Reply:
I'll change it now!